src/guiguts/search.py

```diff
@@ -57,9 +57,9 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
         except AttributeError:
             SearchDialog.selection = tk.BooleanVar(value=False)
 
-        kwargs["resize_y"] = False
+        # kwargs["resize_y"] = False
         super().__init__("Search & Replace", *args, **kwargs)
-        self.minsize(400, 100)
+        # self.minsize(400, 100)
```

Comment:
Re-commenting line 60 and uncommenting line 62 has the same effect as having both of the commented out. That is, the S&R dialog retains its size and placement from the last time it was opened.

Note that in either case (i.e. either with both 60 and 62 commented out, or with only 60 commented out, the y-axis is now modifiable.